google-cloud-core/lib/google/cloud/config.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
We have avoided the magic frozen string comment in this gem so far. Adding it seems unrelated to the Ruby 2.7 compatibility fix.

Reply:
Indeed, it was an opportunistic change to prevent `end_with?("=")` from allocating a new string on every call. I can remove it if you'd like.

I say it across the project so I assume it was ok to add it.